### PR TITLE
add dial-out to Zone A countries or regions policy

### PR DESCRIPTION
**Pull request by fafan**
@tonysmit please review.

I also want to add "([Learn more](https://docs.microsoft.com/en-US/microsoftteams/complimentary-dial-out-period))" after the option Zone A countries or regions only. But I don't know how to add a link there....Could you help with that?

Thanks,

Fan

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/outbound-calling-restriction-policies.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Outbound call controls can be applied on a per-user basis and provide the follow
 
 |Control|Description|Control options|
 |:-----|:-----|:-----|
-|Audio Conferencing PSTN calls|Restricts the type of outbound </br>calls that are allowed from within </br>meetings organized by a user.|International and Domestic (default)</br>Domestic</br>None|
+|Audio Conferencing PSTN calls|Restricts the type of outbound </br>calls that are allowed from within </br>meetings organized by a user.|Any destination (default)</br>In the same country or region as the orgnizor </br> </br>Zone A countries or regions only </br>Don't allow|
```
Comment (JohanFreelancer9) [inline Teams/outbound-calling-restriction-policies.md:34]:
```suggestion
|Audio Conferencing PSTN calls|Restricts the type of outbound </br>calls that are allowed from within </br>meetings organized by a user.|Any destination (default)</br>In the same country or region as the organizer</br> </br>Zone A countries or regions only </br>Don't allow|
```

File: Teams/outbound-calling-restriction-policies.md [hunk 2]
```diff
@@ -88,3 +88,6 @@ The following table provides an overview of each policy.
 |    Identity='tag:DialoutCPCDomesticPSTNDisabled'    |    User in the conference can only dial out to   domestic numbers, and this user cannot make any outbound calls to PSTN number besides emergency numbers.    |
 |    Identity='tag:DialoutCPCDisabledPSTNDomestic'    |    User in the conference cannot make any dial   out, and this user can only make outbound call to domestic PSTN numbers.    |
```
Comment (JohanFreelancer9) [inline Teams/outbound-calling-restriction-policies.md:89]:
```suggestion
|    Identity='tag:DialoutCPCDisabledPSTNDomestic'    |    User in the conference cannot make any dial   out, and this user can only make outbound calls to domestic PSTN numbers.    |
```

File: Teams/outbound-calling-restriction-policies.md [hunk 3]
```diff
@@ -88,3 +88,6 @@ The following table provides an overview of each policy.
...
 |    Identity='tag:DialoutCPCandPSTNDisabled'    |    User in the conference cannot make any dial   out, and this user cannot make any outbound calls to PSTN number besides emergency numbers.    |
+|    Identity='tag:DialoutCPCZoneAPSTNInternational'    |    User in the conference can only dial out to Zone A countries and regions, and this user can make outbound calls to international and domestic numbers.    |
+|    Identity='tag:DialoutCPCZoneAPSTNDomestic'    |    User in the conference can only dial out to Zone A countries and regions, and this user can only make outbound calls to domestic PSTN number.    |
```
Comment (JohanFreelancer9) [inline Teams/outbound-calling-restriction-policies.md:92]:
```suggestion
|    Identity='tag:DialoutCPCZoneAPSTNDomestic'    |    User in the conference can only dial out to Zone A countries and regions, and this user can only make outbound calls to domestic PSTN numbers.    |
```

File: Teams/outbound-calling-restriction-policies.md [hunk 4]
```diff
@@ -88,3 +88,6 @@ The following table provides an overview of each policy.
...
+|    Identity='tag:DialoutCPCZoneAPSTNDisabled'    |    User in the conference can only dial out to Zone A countries and regions, and this user cannot make any outbound calls to PSTN number besides emergency numbers.    |
```
Comment (JohanFreelancer9) [inline Teams/outbound-calling-restriction-policies.md:93]:
```suggestion
|    Identity='tag:DialoutCPCZoneAPSTNDisabled'    |    User in the conference can only dial out to Zone A countries and regions, and this user cannot make any outbound calls to PSTN numbers besides emergency numbers.    |
```

**Comment by fafan** [comment]
Hi @JohanFreelancer9 , do I need to commit all changes or submit another PR? (sorry, first time updating the doc on Github) Thanks!

File: Teams/outbound-calling-restriction-policies.md (hunk 4 above)
Comment (tonysmit) [reply Teams/outbound-calling-restriction-policies.md:95]: @JohanFreelancer9 @scanum when you are editing please beware that we must always use “country/region” or “country or region” in all documentation. It’s a geopolitical issue that we must comply with. Please let the other vendors know as well please.

Comment (JohanFreelancer9) [reply Teams/outbound-calling-restriction-policies.md:95]: @tonysmit  Thanks for the information. Will do.

File: Teams/outbound-calling-restriction-policies.md (hunk 1 above)
Comment (tonysmit) [reply Teams/outbound-calling-restriction-policies.md:34]: @JohanFreelancer9 - Can you check on the publishing status of this PR please? The PM said the updates aren't available yet. Thanks!

Comment (JohanFreelancer9) [reply Teams/outbound-calling-restriction-policies.md:34]: @scanum Could you please follow up as discussed? Thanks.

Comment (scanum) [reply Teams/outbound-calling-restriction-policies.md:34]: @tonysmit As of my understanding, it should be published now that it has been merged. Please let me know if there is anything I can do to help. Thanks

**Pull request merged**